practicals/large_language_models.ipynb

```diff
@@ -16,7 +16,7 @@
         "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    # TODO: update guidance or remove this cell.
</pre><p>Remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2023/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>